core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/implementation/MultiSerialiser.java

```diff
@@ -62,15 +63,22 @@
  *         },
  *         "valueClass" : "java.lang.String"
  *       }, {
+ *         "key" : 7,
+ *         "serialiser" : {
+ *           "class" : "old.depricated.and.deleted.IntegerSerialiser",
+ *           "charset" : "ISO-8859-1"
+ *         },
+ *         "valueClass" : "java.lang.Integer"
+ *       }, {
  *         "key" : 24,
  *         "serialiser" : {
- *           "class" : "uk.gov.gchq.gaffer.serialisation.implementation.raw.CompactRawIntegerSerialiser"
+ *           "class" : "uk.gov.gchq.gaffer.serialisation.implementation.ordered.OrderedIntegerSerialiser"
  *         },
  *         "valueClass" : "java.lang.Integer"
  *       }, {
  *         "key" : 8,
  *         "serialiser" : {
- *           "class" : "uk.gov.gchq.gaffer.serialisation.implementation.ordered.OrderedIntegerSerialiser"
+ *           "class" : "uk.gov.gchq.gaffer.serialisation.implementation.raw.CompactRawIntegerSerialiser"
```

Comment:
assumption that this is still ordered because that was the reason for removing the old `RawIntegerSerialiser`.